pkg/oidcprovider/oidcprovider.go

```diff
@@ -107,6 +109,10 @@ func MountRoutes(s *service.Service) error {
 			"sub",
 			"name", "family_name", "given_name",
 			"email", "email_verified",
+			"pls_*",
+		},
+		SupportedScopes: []string{
+			"openid", "profile", "email", "phone", "address", "offline_access", "pls_*",
```

Comment:
This is probably just me not knowing enough of OIDC, but do we _need_ to specify `phone` and `address` here? It's not stuff we have or want to have

Reply:
its probably best then. And of course we can change it in the future